examples/github-complete/terraform.tfvars.sample

```diff
@@ -3,3 +3,6 @@ alb_ingress_cidr_blocks = ["x.x.x.x/32"]
 github_owner = "myorg"
 github_user = "atlantis"
```

Comment:
user and token would be removed here as well

Reply:
@calexandre

Reply:
I cannot remove this because it would break the repository webhook creation.